Open Judge System/Data/OJS.Data/Repositories/SubmissionsRepository.cs

```diff
@@ -1,4 +1,7 @@
-﻿namespace OJS.Data.Repositories
+﻿using System.Collections.Generic;
+using System.Runtime.Remoting.Messaging;
```

Comment:
usings should be nested in namespace and separated by newline per source assembly, i.e 
namespace .... 
using System;
using System.Linq;
_newline_
using OJS.Data.Models;
using OJS.Data.Repositories.Base;
etc.